src/main/java/org/fundacionjala/at15/katas/pokerhands/vladimir/Card.java

```diff
@@ -0,0 +1,63 @@
+package org.fundacionjala.at15.katas.pokerhands.vladimir;
+
+public class Card extends Deck {
```

Comment:
Not really sure about this logic, a "Deck" would be a group of cards, so between Card and Deck there should be a composition relationship rather than inheritance

Reply:
Yes, you are right. I will correct that, thank you Jose